README.md

```diff
@@ -15,12 +15,59 @@ or for the Home Office.
 
 Almost all HOCS code is open-source and publicly available on GitHub.
 
-All HOCS code on GitHub is hosted within the UKHomeOffice organisation.
-You can [search the organisation](https://github.com/UKHomeOffice/?q=hocs) for
-a list of our repositories. Some repositories are archived, and no longer in use.
+### Core components
+[hocs-audit](https://github.com/UKHomeOffice/hocs-audit)
+
+[hocs-extracts](https://github.com/UKHomeOffice/hocs-extracts)
+
+[hocs-case-creator](https://github.com/UKHomeOffice/hocs-case-creator)
+
+[hocs-casework](https://github.com/UKHomeOffice/hocs-casework)
+
+[hocs-data 🔒️ *(deprecated)*](https://github.com/UKHomeOffice/hocs-data)
+
+[hocs-data-wcs 🔒️ *(deprecated)*](https://github.com/UKHomeOffice/hocs-data-wcs)
+
+[hocs-docs](https://github.com/UKHomeOffice/hocs-docs)
+
+[hocs-docs-converter](https://github.com/UKHomeOffice/hocs-docs-converter)
+
+[hocs-info-service *(deprecated)*](https://github.com/UKHomeOffice/hocs-info-service)
```

Comment:
To avoid confusion should we mark this as "To be deprecated"?

Reply:
Is this to be deprecated - when we discussed some referential data makes sense in it.

Reply:
you'd build a reference data service differently to hocs-info-service, shall I just remove the all brackets?

Reply:
Just remove this one from hocs-info-service till we actually deprecate.